MobileWallet/UIElements/EmoticonView.swift

```diff
@@ -213,62 +209,56 @@ class EmoticonView: UIView {
 
     func expand(_ expand: Bool, completion: (() -> Void)? = nil, callTapCompletion: Bool = false, animated: Bool = true) {
```

Comment:
I think it's better if separate it into two methods: expand and shrink maybe? @SerhiiShovkoplias

Reply:
I do not think it's a good idea to create two methods with a difference of only 1 boolean parameter. And call same methods.